helm/kyverno-policies-dx/templates/DeprecatedAPI.1.25.yaml

```diff
@@ -0,0 +1,36 @@
+# THIS FILE IS GENERATED WITH 'make generate' - DO NOT EDIT MANUALLY
+{{- if .Capabilities.KubeVersion < 1.25 }}
```

Comment:
Not sure if this is the right sintax tho, have you checked it? I just threw it out of the blue without verifying 😅

Reply:
Updated to semverCompare